package.json

```diff
@@ -39,19 +39,19 @@
     "typescript": "^2.6.2"
   },
   "dependencies": {
-    "@types/bluebird": "3.5.18",
+    "@types/bluebird": "3.5.20",
     "@types/dockerode": "2.5.5",
-    "@types/lodash": "^4.14.87",
-    "@types/tar-stream": "^1.6.0",
+    "@types/lodash": "4.14.112",
```

Comment:
Why did you remove the `^`?

Reply:
Because of this pull request from @Page- : https://github.com/balena-io/resin-builder/pull/502
Is this best practice or not?...